velocidi-sdk/build.gradle

```diff
@@ -51,7 +54,6 @@ android {
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation 'com.android.volley:volley:1.1.1'
```

Comment:
This is an HTTP library. Is this delete intentional?

Reply:
Yes. Previously we were using `Volley` as our main HTTP library. In https://github.com/velocidi/velocidi-android-sdk/pull/7 we moved to `okhttp` but I forgot to remove the dependency.